logging.conf

```diff
@@ -0,0 +1,27 @@
+[loggers]
```

Comment:
I guess we could get rid of root altogether; unless there are strong counter-indications